### PR TITLE
Fix Android crash for mobile notifications screen

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Fixes https://github.com/mattermost/mattermost-mobile/issues/1109

### CODE REVIEW

File: app/screens/settings/notification_settings/notification_settings.js [hunk 1]
```diff
@@ -226,6 +228,42 @@ class NotificationSettings extends PureComponent {
...
+            value: sendImmediatleyValue,
+            checked: sendImmediatley
+        }];
+
+        if (emailBatchingEnabled) {
+            emailOptions.push({
+                label: intl.formatMessage({
+                    id: 'user.settings.notifications.email.everyXMinutes',
+                    defaultMessage: 'Every {count, plural, one {minute} other {{count, number} minutes}}'
+                }, {count: Preferences.INTERVAL_FIFTEEN_MINUTES / 60}),
```
Comment (mkraft) [inline app/screens/settings/notification_settings/notification_settings.js]: `Preferences.INTERVAL_FIFTEEN_MINUTES / 60`. So every 15 seconds?

Comment (mkraft) [reply app/screens/settings/notification_settings/notification_settings.js]: Oh I guess this was there before now that I scroll down.

**Comment by hmhealey** [comment]
Code looks good, but what did you find to be the cause of that crash?

**Comment by enahum** [comment]
Looks like it was the mapping on the children that was causing something really weird sometimes it worked others it didn’t so I remove the children and added an array instead

**Pull request merged**